docs/get-details/dapps/smart-contracts/frontend/apps.md

```diff
@@ -226,11 +225,13 @@ Use the creator_mnemonic to define sender:
 
 === "Python"
     <!-- ===PYSDK_ACCOUNT_RECOVER_MNEMONIC=== -->
-	```python
-    # get account from mnemonic
-    private_key = mnemonic.to_private_key(creator_mnemonic)
-    sender = account.address_from_private_key(private_key)
-    ```
+```python
+mn = "cost piano sample enough south bar diet garden nasty mystery mesh sadness convince bacon best patch surround protect drum actress entire vacuum begin abandon hair"
```

Comment:
https://testnet.algoexplorer.io/address/C4TKA332MLC6HLVFWWF3LXG3WXKAMNJDZMDE6SBSROO3KEIECBFD5FJ7QM

So stupid question, but should we designate a particular account to use in the docs or remove it and leave instructions to populate it instead?

Reply:
Good question, I think a single mnemonic for examples that need a mnemonic is probably the right call but lets see what @JasonWeathersby thinks